README.md

```diff
@@ -20,9 +20,11 @@ The extensions will declare a new module, exposed to Lua as `screenshot`. The ex
 		screenshot.html5(function(self, base64_img)
 			-- share your base64 png
 		end)
+		-- Capture screenshots of a portion of the screen:
+		-- screenshot.html5(callback, x, y, width, height)
```

Comment:
I think I'd actually rather prefer screenshot.html5([x, y, width, height], callback) and a runtime check to parse the arguments. What do you think about that?

Reply:
done